static/docs/commands-reference/diff.md

```diff
@@ -54,10 +86,50 @@ Git reference.
 
 ```dvc
     $ dvc diff HEAD^
-    dvc diff from 6a819f8fa053f124f6a5487efc824a8c17366c71 to 612f6caf5c5daeb172167db285efd8b169d41b60
+    dvc diff from df613bce6dc0738f71c62d1748e1edfb3b7e4893 to ed10968bcb0dab72563d05712f24ddfff698c87b
 
     diff for 'data/data.xml'
     +data/data.xml with md5 a304afb96060aad90176268345e10355
 
     added file with size 37.9 MB
 ```
+
+## Examples: Specific target accross Git references
+
+We can base this example in the [Experiment Metrics](/doc/get-started/metrics)
+and [Compare Experiments](/doc/get-started/compare-experiments) sections of our
+Get Started guide, which describe different experiments to produce the
+`model.pkl` file.
+> Our sample repository has the `bigrams-experiment` and `baseline-experiment`
+[tags](https://github.com/jorgeorpinel/example-get-started/tags) respectively to
+refernce these experiments.
+
+### Click and expand to setup example
+
+Having followed the previous example's setup, move into the
+**example-get-started** directory. Then make sure that you have the latest code
+and data with the following commands.
+
+```dvc
+    $ git checkout master
+    $ dvc fetch -aT
+```
+
+The `-aT` flag passed to `dvc fetch` makes sure we have all the data files
+related to all existing tags in the repo. You take a look at the available tags
+of our sample repo in https://github.com/jorgeorpinel/example-get-started/tags.
+
+</details>
+
+```dvc
+    $ dvc diff -t model.pkl baseline-experiment bigrams-experiment
+    dvc diff from bc1722d7eeb4cba9a5c8e401199e995739c474a9 to 8c1169d1819c5cf0a4e2aa7e7d8c43854563b251
+
+    diff for 'model.pkl'
+    -model.pkl with md5 a66489653d1b6a8ba989799367b32c43
+    +model.pkl with md5 3863d0e317dee0a55c4e59d2ec0eef33
+    ...
+````
+
+The output from this command confirms that there's a difference in the
+`model.pkl` file between the 2 Git references we indicated.
```

Comment:
let's also add an example that compares directories?

Reply:
Will do.

Reply:
Ready, please check last example called "Compare a directory"